inc/spbc-admin.php

```diff
@@ -150,12 +150,12 @@ function spbc_admin_init()
     }
 
     // Drop debug data
-    if (Post::get('spbc_debug__drop')) {
+    if (Post::get('spbc_debug__drop', null, 'word')) {
```

Comment:
Use ::getString() - this method works like get, but return exact value type

https://github.com/CleanTalk/wordpress-antispam/blob/baade312298c3d91b8c18b7632ff0cb9def96903/lib/Cleantalk/Variables/ServerVariables.php#L65

Reply:
Thanks, fixed https://github.com/CleanTalk/security-malware-firewall/pull/492/commits/6b12e35a1534a8c212021ba02fdf7fa6ca059372

Please recheck